.buildkite/packaging.pipeline.yml

```diff
@@ -265,6 +334,76 @@ steps:
         artifact_paths:
           - build/distributions/**/*
 
+      - label: "STAGING: {{matrix}} FIPS"
+        env:
+          PLATFORMS: "${PLATFORMS_FIPS}"
```

Comment:
```suggestion
          PLATFORMS: "${PLATFORMS_AMD64_FIPS}"
```

Reply:
Fixed in e1cf0bd091e854c0a8a936fe6dfb234c0cea98ad